tests/app/mocked_services.py

```diff
@@ -0,0 +1,30 @@
+import json
+from frontstage import app
+
+with open('tests/test_data/party/business_party.json') as json_data:
```

Comment:
:+1: Like this a lot, This could tidy up a lot of the tests across quite a few repos